server/form-pages/apply/risks-and-needs/health-needs/mentalHealth.ts

```diff
@@ -104,10 +99,13 @@ export default class MentalHealth implements TaskListPage {
       delete this.body.servicesDetail
     }
 
-    if (this.body.hasPrescribedMedication !== 'yes') {
-      delete this.body.isInPossessionOfMeds
-      delete this.body.medicationDetail
+    if (this.body.canManageMedication !== 'yes') {
```

Comment:
do we need a condition here to check if the answer is changed to the third option,`They are not prescribed medication for their mental health` ?

Reply:
oh my bad, I realise it's covered by checking it's neither 'yes' or 'no'